l5kit/l5kit/tests/evaluation/metrics_test.py

```diff
@@ -1,26 +1,45 @@
 import numpy as np
 import pytest
 
-from l5kit.evaluation.metrics import neg_multi_log_likelihood
-
-
-def test_neg_multi_log_likelihood_shapes() -> None:
-    gt = np.random.randn(12, 2)
-    pred = np.random.randn(4, 12, 2)
-    avail = np.ones(12)
-    conf = np.random.rand(4)
+from l5kit.evaluation.metrics import (
+    _assert_shapes,
+    average_displacement_error_mean,
+    average_displacement_error_oracle,
+    final_displacement_error_mean,
+    final_displacement_error_oracle,
+    neg_multi_log_likelihood,
+    prob_true_mode,
+    rmse,
+    time_displace,
+)
+
+
+def test_assert_shapes() -> None:
+    num_modes, future_len, num_coords = 4, 12, 2
+
+    gt = np.random.randn(future_len, num_coords)
+    pred = np.random.randn(num_modes, future_len, num_coords)
+    avail = np.ones(future_len)
+    conf = np.random.rand(num_modes)
```

Comment:
Do we need the rand / randn above? Could we just use zeros? From a quick read it seems like it shouldn't make a difference. Just a bit worried of using randomness in tests as they may end up flaky.